src/main/java/ddeliopoulos/github/gardenjournal/journalentry/JournalController.java

```diff
@@ -0,0 +1,44 @@
+package ddeliopoulos.github.gardenjournal.journalentry;
+
+import ddeliopoulos.github.gardenjournal.journalentry.api.CreateJournalRequest;
+import ddeliopoulos.github.gardenjournal.journalentry.api.GetJournalResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "/journal-entries")
+public class JournalController {
+
+    private final JournalService journalService;
+
+    @Autowired
+    public JournalController(JournalService journalService) {
+        this.journalService = journalService;
+    }
```

Comment:
[nitpick] Could be replaced with [Lombok's `@RequiredArgsConstructor`](https://projectlombok.org/features/constructor) ⁽ᵃˡˢᵒ ˢᵉᵉ [ᵒᵘʳ ᵍᵘᶦᵈᵉˡᶦⁿᵉˢ](https://github.com/Glovo/backend-guidelines/blob/master/guidelines/lombok.md#requiredargsconstructor) ᶠᵒʳ ᵐᵒʳᵉ ᶦⁿᶠᵒ ᵒⁿ ʰᵒʷ ᵗʰᶦˢ ᵃⁿⁿᵒᵗᵃᵗᶦᵒⁿ ᶦˢ ᵘˢᵉᵈ ᶦⁿ ᵗʰᵉ ᵇᵃᶜᵏ⁻ᵉⁿᵈ⁾.

ᴛʜɪs ᴀᴄᴛɪᴏɴ ᴡᴀs ᴘᴀʀᴛɪᴀʟʟʏ ᴀᴜᴛᴏᴍᴀᴛᴇᴅ ʙʏ [GɪᴛHᴜʙ's ʀᴇsᴘᴏɴsᴇ ᴛᴇᴍᴘʟᴀᴛᴇs](https://github.com/settings/replies)

Reply:
I fixed it! :dancers: